akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -2447,6 +2447,7 @@ final class Source[Out, Mat](delegate: scaladsl.Source[Out, Mat]) extends Graph[
    * The resource creation function is invoked once when the stream is materialized and the returned resource is passed to
    * the mapping function for mapping the first element. The mapping function returns a mapped element to emit
    * downstream. The returned `T` MUST NOT be `null` as it is illegal as stream element - according to the Reactive Streams specification.
+   * A `null` ressource is not allowed and will fail the stream.
```

Comment:
```suggestion
   * A `null` resource is not allowed and will fail the stream.
```
Here and in more places